lib/tapioca/generators/base_generator.rb

```diff
@@ -0,0 +1,110 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "active_support/all"
+
+module Tapioca
+  module Generators
+    class BaseGenerator
```

Comment:
Do we need to repeat `Generator` here? 

```suggestion
    class Base
```

For example in Rails most base class are just called `Base`, the namespace being enough to disambiguate.